cmd/influx_inspect/cardinality/aggregators/aggregators.go

```diff
@@ -0,0 +1,246 @@
+package aggregators
+
+import (
+	"fmt"
+	"strings"
+	"sync"
+	"text/tabwriter"
+
+	"github.com/influxdata/influxdb/cmd/influx_inspect/report"
+	"github.com/influxdata/influxdb/models"
+)
+
+type rollupNodeMap map[string]RollupNode
```

Comment:
nitpick: not sure you get anything from defining this struct vs. just using `map[string]RollupNode`? Not necessary to fix.